Sources/SourceKitLSP/SourceKitLSPServer+Options.swift

```diff
@@ -49,6 +49,9 @@ extension SourceKitLSPServer {
     /// notification when running unit tests.
     public var swiftPublishDiagnosticsDebounceDuration: TimeInterval
 
+    /// Experimental features that are enabled.
+    public var experimentalFeatures: [ExperimentalFeature]
```

Comment:
Any reason not to make a `Set`?

Reply:
Yeah I wasn't really thinking of performance, more just semantically it feels like `Set` better fits. But I don't feel strongly at all 🤷

Reply:
Neither do I. It’s in as an array now, let’s just leave it.